examples/flutter_gallery/ios/.gitignore

```diff
@@ -36,3 +36,5 @@ Icon?
 /Flutter/Flutter.framework
 /Flutter/Generated.xcconfig
 /ServiceDefinitions.json
+
+Pods/
```

Comment:
Thanks! Wondered if that might be the case -- github doesn't allow expanding collapsed regions on phones :(

(and thanks for making this change!)